src/config.ts

```diff
@@ -360,11 +361,44 @@ export const getHttpsApiFetchWithBearer = (bearer: string) =>
     )
   );
 
+export const MessagesFeatureFlagType = t.union([
```

Comment:
https://github.com/gcanti/io-ts/tree/1.x#union-of-string-literals

Reply:
Thanks! Done in [f0c9eca](https://github.com/pagopa/io-backend/pull/882/commits/f0c9ecac61390c067788b47867ae612207dbd68c)